migrations/versions/0336_broadcast_msg_content_2.py

```diff
@@ -11,30 +11,14 @@
 from sqlalchemy.dialects import postgresql
 from sqlalchemy.orm.session import Session
 
-from app.models import BroadcastMessage
```

Comment:
It looks like this was never used. I'm wondering if we should have left all of the migrations as is (and then add one to remove everything) and just deleted this one line to deal with the model not existing anymore